### ISSUE TITLE
[YouTube Downloader] ...

### ISSUE THREAD

**Issue by khlebobul** [raycast/extensions#9051]
### Extension

https://www.raycast.com/vimtor/youtube-downloader

### Description

I've been unable to narrow down videos through this extension for a few days now. I get the same error every time. How can this problem be solved?

### Steps To Reproduce

1. Copy video link
2. Paste into extension
3. Waiting loading
4. Getting error 

### Current Behaviour

This is the error I'm getting `Encoding Failed ffmpeg was killed with signal SIGABRT`

### Expected Behaviour

Uploading a video

**Comment by vimtor** [comment]
Hi @khlebobul!

When you say "narrow down video," do you mean setting the start and end time values?

![CleanShot 2023-11-04 at 19 43 02@2x](https://github.com/raycast/extensions/assets/36263538/42c66e56-d920-489e-8493-cdc6d29bda2d)

Can you please share a video URL and the values you are setting in those inputs?

Thank you!

**Comment by khlebobul** [comment]
Hello, yes, I can't just upload a video. It looks like this every time:

https://github.com/raycast/extensions/assets/77191581/60fb6e16-f875-48f7-9e20-e82ec0735d2e

[Design Tips to Convert More Customers | Design Review](https://www.youtube.com/watch?v=tJTxDqjO4vg&list=WL&index=11)

**Comment by vimtor** [comment]
The unofficial YouTube API can be quite flaky sometimes. However, it should work most of the time. Also, in this case, it is failing in the download process, not encoding the video with `ffmpeg`, so I don't have enough information here.

I will implement a copy/paste error feature so you can paste the error here, and we can better debug the issue.

**Comment by khlebobul** [comment]
`Encoding Failed ffmpeg was killed with signal SIGABRT`

**Comment by vimtor** [comment]
Oh yeah, I didn't read the original issue properly. Could you please tun `which ffmpeg` in your terminal and make sure it is set to the same path in the Raycast extension? Same thing with `ffmprobe`.

Also if you could print here those outpus + the output of running `ffmpeg -v`,

**Comment by khlebobul** [comment]
- `which ffmpeg` ->  /opt/homebrew/bin/ffmpeg
- `which ffprobe` -> /opt/homebrew/bin/ffprobe

I didn't understand message about `ffmpeg -v`, it gave me error 👇. Maybe that's why I have error 
<img width="1225" alt="SCR-20231111-obtk" src="https://github.com/raycast/extensions/assets/77191581/2419d525-5b97-4387-8ab2-282ad9e81c41">

**Comment by vimtor** [comment]
Yeah, that seems to be the issue. Could you please run `brew reinstall ffmpeg`?

If that doesn't work maybe you can do:

```
brew uninstall mbedtls
brew install mbedtls
sudo brew link mbedtls
```

Not sure if it would work but I found the solution [here](https://superuser.com/questions/877510/ffmpeg-dyld-library-not-loaded-usr-local-lib-libmp3lame-0-dylib).

**Comment by khlebobul** [comment]
Unfortunately, it didn't work. I even tried reinstalling the extension to make sure the file path was correct

**Comment by vimtor** [comment]
After doing the `brew uninstall`, `install` and `link`, do you still encounter the same error?

**Comment by khlebobul** [comment]
Yes, same error. How this extension works on your mac? Is it fine?

**Comment by vimtor** [comment]
Yes! It works fine, this is the first time this issue has been reported.

Maybe you can try [this solution](https://stackoverflow.com/questions/77110765/error-while-run-command-ffmpeg-library-not-loaded-opt-homebrew-opt-mbedtls-l):

```
brew uninstall librist --ignore-dependencies
brew uninstall mbedtls --ignore-dependencies
brew reinstall ffmpeg
```

**Comment by khlebobul** [comment]
Sorry, but same error

**Comment by vimtor** [comment]
The only solution left would be to [install `ffmpeg` manually](https://bbc.github.io/bbcat-orchestration-docs/installation-mac-manual/). I've never done that, so I cannot assure you that it will work.

**Comment by khlebobul** [comment]
Thank you! I'll try

**Comment by arkulpaKonstantin** [comment]
I encounter the same issue and also reinstalling ffmpeg did not fix it. Did not try manually installing it.
